### DISCUSSION TITLE
Installation on separate schema possible?

### DISCUSSION THREAD

**Discussion by rnijs** [flowsforapex/apex-flowsforapex#593] [Q&A]
Hi,

I just want to know if there is a possibility to install this nice project into a separate workspace and database schema as we do not want to mix this with all our own development. I suppose we also need to grant API access, and install the plugins into our own workspace.

Are there any notes or guidelines on this?

Thanks for your help.
Ronny

**Comment by rnijs** [comment]
Did someone already set up this in a separate schema? Thanks for the information.

**Comment by diegobkc** [comment]
Was there ever a resolution to this issue? Does Flows for APEX need to be installed in the same schema as the application you are trying to use it with?
